alfredo-zimperz/challenges/to-do-redux/src/components/Dashboard/index.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useEffect } from 'react';
+import { connect } from 'react-redux';
+import { PropTypes } from 'prop-types';
+import { loadBooks } from '../../actions/actionCreators';
+
+const Dashboard = ({ books, dispatch }) => {
+  useEffect(() => {
+    loadBooks().then((updatedBooks) => {
+      console.log(updatedBooks);
+      dispatch(updatedBooks);
+    });
+  }, []);
+
+  const title = 'Dashboard';
+  return (
+    <div>
+      <h2>{title}</h2>
+      <ul>
+        {
+        (books) && (
+          books.map((book) => (
+            <li>
```

Comment:
You should add a key for rendering components in loop. In this example you could do:

`<li key={book.id} >`